src/domain/project/project.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { InputType, Field } from '@nestjs/graphql';
+import { MaxLength } from 'class-validator';
+
+@InputType()
+export class ProjectInput {
```

Comment:
Not sure it's supposed to be here but we're missing the **textId** and **shortName**? Also are we going to introduce duration for a project in this version?